### PR TITLE
add log

### PR DESCRIPTION
**Pull request by MingshiPeng**
Add log functionality.
screen shot 2018-04-30 at 9 35 06 pm

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ def index():
 @app.route('/inventory', methods=[GET])
 def query_prod_info():
     """ Query specific entries in the Inventory system """
+    app.logger.info("GET received, List all.")
```
Comment (majia67) [inline app/server.py]: This route also responses to the query API. Please also log the specific query conditions

**Review by majia67 — CHANGES_REQUESTED** [review]

File: app/server.py (hunk 1 above)
Comment (MingshiPeng) [reply app/server.py]: Addressed.
screen shot 2018-05-01 at 11 16 30 am

**Review by majia67 — APPROVED** [review]
LGTM

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-185525 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-185525") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit 8497cc37d7593396e00f095159e3b99af1db5710

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-185835 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-185835") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit 8497cc37d7593396e00f095159e3b99af1db5710

**Pull request merged**